go.mod

```diff
@@ -46,13 +46,15 @@ require (
 	k8s.io/client-go v0.24.7
 	k8s.io/klog/v2 v2.70.1
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	oras.land/oras-go v1.2.1
```

Comment:
for all the new dependencies being pulled here, would you mind linking their license in the PR description? I took a stab at that here: https://github.com/open-policy-agent/gatekeeper/pull/2374 to make it easier on reviewers to check licenses

Reply:
TBH what we really need is to automatically check licenses, relying on humans to do this is not going to be tenable. Out of scope for this PR, though.

Reply:
K8s's license verification script: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-licenses.sh

Reply:
Added #2404